app/models/user.rb

```diff
@@ -4,6 +4,7 @@ class User < ApplicationRecord
 
   belongs_to :mobile_network, optional: true
   belongs_to :responsible_body, optional: true
+  has_many :key_responsible_bodies, class_name: 'ResponsibleBody', foreign_key: :key_contact_id, inverse_of: :key_contact
```

Comment:
so a User can only `belong_to` one ResponsibleBody, but can be key_contact for many RBs? Is that right?
mixing up the one/many like that makes me nervous

Do we even need this relationship on user?

Reply:
Yes a bit confusing, I don't __think__ we need it, just thought it might be useful. I'll remove it.